README.md

```diff
@@ -22,7 +22,7 @@ The project organization has been designed to maximise maintainability and acess
   - `107_calculus` (Course code and name, refrain from using abbreviations)
     - `reference_material` (textbooks, papers, etc.)
     - `ashok_kumar` (Faculty name, refrain from using abbreviations)
-      - `2016` (Year)
+      - `2016` (Year in which you have taken the course)
```

Comment:
Suggest changing this to `2016 (Year in which the course was offered)`

Reply:
Courses can be offered in multiple years mate......it makes more sense to ask about when **you** have taken the course right?

Reply:
You're right, I thought it would be implicit that it addresses the contributor. But we've seen what happened with the implicit `year` thing. Better to retain it this way.